src/main/java/com/linkedin/kmf/services/CommitLatencyMetrics.java

```diff
@@ -41,14 +42,47 @@
     _commitOffsetLatency.add(new MetricName("commit-offset-latency-ms-avg", METRIC_GROUP_NAME, "The average latency in ms of committing offset", tags), new Avg());
     _commitOffsetLatency.add(new MetricName("commit-offset-latency-ms-max", METRIC_GROUP_NAME, "The maximum latency in ms of committing offset", tags), new Max());
 
+    if (latencyPercentileGranularityMs == 0) {
+      throw new IllegalArgumentException();
+    }
     int bucketNum = latencyPercentileMaxMs / latencyPercentileGranularityMs + 2;
```

Comment:
Can you add a comment here explaining why `+ 2` ?

Reply:
Done